SPECS-EXTENDED/python-pyroute2/python-pyroute2.spec

```diff
@@ -0,0 +1,233 @@
+%global srcname pyroute2
+
+%{!?python3_pkgversion:%global python3_pkgversion 3}
+
+# FIXME(hguemar): Fix for EL7, in summary
```

Comment:
Can we remove references to Fedora, RHEL, etc. that don't make sense in a Mariner context?